src/storybook/pages/home/home--logged-in.stories.tsx

```diff
@@ -79,7 +79,9 @@ const Template = (args): JSX.Element => (
         <sbb-menu-action>Deutsch</sbb-menu-action>
         <sbb-menu-action>Français</sbb-menu-action>
         <sbb-menu-action>Italiano</sbb-menu-action>
```

Comment:
I guess it's better to mark the others with false, so a user hears that he can change it
```suggestion
        <sbb-menu-action aria-pressed="false">Deutsch</sbb-menu-action>
        <sbb-menu-action aria-pressed="false">Français</sbb-menu-action>
        <sbb-menu-action aria-pressed="false">Italiano</sbb-menu-action>
```